frontend/src/app/models/Absence.model.ts

```diff
@@ -1,19 +1,41 @@
 import { User } from './User.model';
+import { DataEntity } from './DataEntity';
 
-export class Absence {
-    constructor(
-        public id: number,
-        public createdAt: Date,
-        public summary: number,
-        public begin: Date,
-        public end: Date,
-        public reporter: User,
-        public assignee: User,
-        public status: string,
-        public type: string
-        
+//TODO enum names imported from backend, we should consider the names as well
+export enum AbsenceType {
```

Comment:
whats the BE model for enums? Bear in mind that if you call console.log(AbsenceType.ABSENCE) you'll get a number (0) for AbsenceType.NON_WORKING you'll get 1. Is this what the BE gives you?